acceptance/invoke/pack_fixtures.go

```diff
@@ -79,6 +79,25 @@ func (m PackFixtureManager) TemplateFixtureToFile(name string, destination *os.F
 	m.assert.Nil(err)
 }
 
+func (m PackFixtureManager) TemplateFile(file *os.File, data map[string]interface{}) {
+	m.testObject.Helper()
+
+	outputTemplate, err := ioutil.ReadAll(file)
+	m.assert.Nil(err)
+
+	_, err = file.Seek(0, 0)
+	m.assert.Nil(err)
+
+	err = file.Truncate(0)
+	m.assert.Nil(err)
```

Comment:
What do these do, together?

Reply:
They delete all the files contents so that the order of operations in the `TemplateFile` function is 
1) read the file's contents into memory
2) delete them
3) re-write out the newly templated contents to the file. 

Note this is only done on files that have been moved into a `tmp` directory for testing